src/test/java/com/google/javascript/gents/singleTests/goog_scope.js

```diff
@@ -0,0 +1,27 @@
+goog.provide("A.B");
+goog.provide("A.B.C");
+
+goog.scope(function() {
+
```

Comment:
Good point.

Unfortunately, this breaks the "aggressive" exporting.  I'd like to leave this as a TODO for a future cl, if possible.